int level
---
ergodic_inspection/Graph graph